requirements/compiled.txt

```diff
@@ -8,4 +8,4 @@ m2crypto==0.22.3
 Pillow==2.8.1
 
 # For the addon validator, including C speedups.
-simplejson==2.3.2
+simplejson>=3.0.5
```

Comment:
since it's atypical for us to use `>=` could you add a comment for why it's needed? Your rationale makes sense though.